### PR TITLE
update instructions for working on ghc-lib

### PR DESCRIPTION
**Pull request by nickchapman-da**
Update instructions for working on ghc-lib.

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ghc-lib/new-instructions.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+
+## Updated instructions for working on `ghc-lib`
+
+Here are instructions for when working on DAML surface syntax, as implemented in the the digital-assert fork of `ghc`. (linked in via `ghc-lib`)
+
+These instructions were written in November 2020, and are intended to replace the [old instructions](working-on-ghc-lib.md), but there might still be useful stuff there.
+
```
Comment (cocreature) [inline ghc-lib/new-instructions.md:7]: I’d just remove this whole section. We shouldn’t merge this without ripping out the old stuff.

**Review by cocreature — COMMENTED** [review]
LGTM thanks! I would just rip out the old stuff completely. We can get it back from the git history if we need it.

File: ghc-lib/new-working-on-ghc-lib.md [hunk 2]
```diff
@@ -1,22 +1,27 @@
 
-## Updated instructions for working on `ghc-lib`
+# Working on `ghc-lib`
 
-Here are instructions for when working on DAML surface syntax, as implemented in the the digital-assert fork of `ghc`. (linked in via `ghc-lib`)
+Copyright 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All Rights Reserved.
```
Comment (cocreature) [inline ghc-lib/new-working-on-ghc-lib.md:4]: copyright at the very beginning please.

File: ghc-lib/new-working-on-ghc-lib.md [hunk 3]
```diff
@@ -1,22 +1,27 @@
...
+SPDX-License-Identifier: (Apache-2.0 OR BSD-3-Clause)
 
-These instructions were written in November 2020, and are intended to replace the [old instructions](working-on-ghc-lib.md), but there might still be useful stuff there.
+If you need to build, test, deploy or develop [`ghc-lib`](https://github.com/digital-asset/ghc-lib) as used by DAML and utilizing the Digital Asset [GHC fork](https://github.com/digital-asset/ghc) these notes are for you.
+
+Here are instructions for when working on DAML surface syntax, as implemented in the the digital-assert fork of `ghc`. (linked in via `ghc-lib`).
+These instructions were updated in November 2020.
```
Comment (cocreature) [inline ghc-lib/new-working-on-ghc-lib.md:10]: That’s what git history is for. This will just get out of date.

**Review by cocreature — APPROVED** [review]
Thanks! 2 minor issues.

**Comment by nickchapman-da** [comment]
Addressed review comments (thanks @cocreature )
And tested the instructions actually worked.

**Pull request merged**